fixtures/showtec/showbar-4.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Showbar 4",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["Kasper Leusink"],
+    "createDate": "2022-06-28",
+    "lastModifyDate": "2022-06-28"
+  },
+  "links": {
+    "manual": [
+      "https://www.liquidx.nl/media/downloads/licht/Showtec%20Showbar4%20%2040329.pdf"
+    ],
+    "productPage": [
+      "https://www.highlite.com/nl/50329-showbar-4.html"
+    ]
+  },
+  "physical": {
+    "dimensions": [1320, 80, 60],
+    "weight": 4.3,
+    "bulb": {
+      "type": "Not included"
+    }
```

Comment:
Is this required?

Reply:
No, better leave out the whole `bulb` section in this case.